pyproject.toml

```diff
@@ -1,10 +1,11 @@
 [build-system]
-requires = ['hatchling']
+requires = ['hatchling>=1.27.0']
 build-backend = 'hatchling.build'
 
 [project]
 name = 'typing-inspection'
 version = '0.4.0'
+license = "MIT"
```

Comment:
```suggestion
license = "MIT"
license-files = ['LICENSE']
```

Reply:
`LICENSE` is one of the default patterns for hatchling. So the file was already included.
Doesn't hurt to be explicit though.

Reply:
Ah wasn't aware of this, good to know